common/src/main/java/org/redlance/dima_dencep/mods/rrls/mixins/LoadingOverlayMixin.java

```diff
@@ -75,9 +84,40 @@ private static int replaceAlpha(int color, int alpha) {
                     minecraft.font, ConfigExpectPlatform.reloadText(), i / 2, 70,
                     ConfigExpectPlatform.rgbProgress() ? ThreadLocalRandom.current().nextInt(0, 0xFFFFFF) : -1
             );
+
+            case Type.TEXT_WITH_BACKGROUND -> {
+                if (rrls$textWidget != null) {
+                    rrls$textWidget.setMaxWidth(i);
+                    rrls$textWidget.setX(i / 2 - rrls$textWidget.getWidth() / 2);
+                    rrls$textWidget.setY(j - j / 3);
+
+                    if (ConfigExpectPlatform.rgbProgress())
+                        rrls$textWidget.setColor(ThreadLocalRandom.current().nextInt(0, 0xFFFFFF));
+
+                    // This will make sure the widget is rendered above other widgets in Pause screen
+                    graphics.pose().pushPose();
+                    graphics.pose().translate(0, 0,255);
+
+                    rrls$textWidget.render(graphics, 0, 0, 0);
+
+                    graphics.pose().popPose();
+                }
+            }
         }
     }
 
+    @Inject(
+            method = "render",
+            at = @At(
+                    value = "INVOKE",
+                    target = "Lnet/minecraft/client/Minecraft;setOverlay(Lnet/minecraft/client/gui/screens/Overlay;)V"
+            )
+    )
+    public void rrls$onLoadDone(GuiGraphics guiGraphics, int mouseX, int mouseY, float partialTick, CallbackInfo ci)
+    {
+        this.rrls$textWidget = null;
```

Comment:
You should also add this for neoforge

Reply:
You can delete this injector, I don't think it's needed

Reply:
> You should also add this for neoforge

It works on neoforge, no need to make another mixin for that:
![2024-08-03 215745](https://github.com/user-attachments/assets/f9f3acec-3c9b-41ea-a079-92c44595a20c)

> You can delete this injector, I don't think it's needed

Ummm... but you just asked to reset after the load completes before?

Reply:
Oh, I remember, it's all right then